webapp/src/components/ProgramActivityDialog.tsx

```diff
@@ -1,85 +1,212 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { decodeHTML } from 'entities';
+import CancelIcon from '@mui/icons-material/Cancel';
+import CloseIcon from '@mui/icons-material/Close';
+import TaskAltIcon from '@mui/icons-material/TaskAlt';
+import LoadingButton from '@mui/lab/LoadingButton';
 import {
-  Button,
+  Alert,
   Dialog,
   DialogActions,
   DialogContent,
   DialogContentText,
   DialogTitle,
   Divider,
+  FormGroup,
+  Snackbar,
   Table,
   TableBody,
   TableCell,
   TableContainer,
   TableRow,
 } from '@mui/material';
-
+import IconButton from '@mui/material/IconButton';
```

Comment:
Can we group this together with the other imports from `@mui/material`? Also, maybe a space after the `} from '@mui/material';` line and before the import from `'../helpers/...'` and then again after the import of `CalendarEvent`.

Reply:
Imports from the icons were grouped together and IconButton import is moved to the @mui/material